derive_builder_core/src/macro_options/darling_opts.rs

```diff
@@ -245,6 +245,9 @@ pub struct Field {
     default: Option<DefaultExpression>,
     #[darling(default)]
     try_setter: Flag,
+    /// Custom builder field type and build method
+    #[darling(default)]
```

Comment:
This is no longer needed; `darling` will automatically handle it because the field is an `Option`.

Reply:
Ah yes.